presto-spi/src/main/java/com/facebook/presto/spi/type/DecimalType.java

```diff
@@ -29,6 +29,7 @@
 {
     public static final int DEFAULT_SCALE = 0;
     public static final int DEFAULT_PRECISION = MAX_PRECISION;
+    public static final DecimalType DECIMAL = createDecimalType(12, 2);
```

Comment:
please undo

Reply:
Types usually have a static created type (e.g. DOUBLE etc), hence I created one for DECIMAL. I will use ``createDecimal`` everywhere instead

Reply:
I was following the usual pattern of types having a static instance which can be returned. I will use ``createDecimal``